### PR TITLE
Added get_kubernetes_versions command and tests

### PR DESCRIPTION
**Pull request by aaron-prindle**
#201

### CODE REVIEW

File: cmd/minikube/cmd/get_kubernetes_versions.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"k8s.io/minikube/pkg/minikube/kubernetes_versions"
+)
+
+// getK8sVersionsCmd represents the ip command
+var getK8sVersionsCmd = &cobra.Command{
+	Use:   "get_kubernetes_versions",
```
Comment (jimmidyson) [inline cmd/minikube/cmd/get_kubernetes_versions.go]: How about just `get-versions`? or `ls-versions`?

File: pkg/minikube/kubernetes_versions/kubernetes_versions.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	return json.NewDecoder(r.Body).Decode(target)
+}
+
+func getK8sVersionsFromURL(url string) (k8sReleases, error) {
+	var k8sVersions k8sReleases
+	if err := getJson(url, &k8sVersions); err != nil {
+		return k8sReleases{}, err
+	}
+	if len(k8sVersions) == 0 {
+		return k8sReleases{}, fmt.Errorf("There were no json k8sReleases at the url specified: %s", url)
```
Comment (dlorenc) [inline pkg/minikube/kubernetes_versions/kubernetes_versions.go]: missing space.

File: pkg/minikube/kubernetes_versions/kubernetes_versions.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+// TODO(aprindle) Change this to correct URL
+//const kubernetesVersionGCSURL = "https://storage.googleapis.com/minikube/k8sReleases.json"
+const kubernetesVersionGCSURL = "https://storage.googleapis.com/minikube-test/k8sReleases.json"
+
+func PrintKubernetesVersionsFromGCS(output io.Writer) {
+	PrintKubernetesVersions(output, kubernetesVersionGCSURL)
+}
+
+// TODO(aprindle) print or return list?
```
Comment (dlorenc) [inline pkg/minikube/kubernetes_versions/kubernetes_versions.go]: Printing is fine here.

**Comment by dlorenc** [comment]
Also please add a k8s_versions.json to deploy/minikube. Also change the test in schema_check.go to check both files.

**Pull request merged**